projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/BaseDataJobDeploymentCrudIT.java

```diff
@@ -65,30 +74,131 @@ public void setup() throws Exception {
   }
 
   @Test
-  public void testDataJobDeploymentCrud() throws Exception {
+  @Order(1)
+  public void uploadJobWithNoUser() throws Exception {
+    mockMvc
+        .perform(
+            post(String.format(
+                    "/data-jobs/for-team/%s/jobs/%s/sources", TEST_TEAM_NAME, testJobName))
+                .content(getJobZipArray())
+                .contentType(MediaType.APPLICATION_OCTET_STREAM))
+        .andExpect(status().isUnauthorized());
+  }
+
+  @Test
+  @Order(2)
+  public void uploadJobWithWrongTeamAndUser() throws Exception {
+    mockMvc
+        .perform(
+            post(String.format(
+                    "/data-jobs/for-team/%s/jobs/%s/sources", TEST_TEAM_WRONG_NAME, testJobName))
+                .with(user("user"))
+                .content(getJobZipArray())
+                .contentType(MediaType.APPLICATION_OCTET_STREAM))
+        .andExpect(status().isNotFound());
+  }
+
+  @Test
+  @Order(3)
+  public void testDataJobDeploymentCrudOperations() throws Exception {
+
+    var testDataJobVersion = uploadJobWithProperUser();
+    var testJobVersionSha = getTestJobVersionSha(testDataJobVersion);
+    var dataJobDeploymentRequestBody = getDataJobDeploymentRequestBody(testJobVersionSha, "3.9");
+    String jobDeploymentName = JobImageDeployer.getCronJobName(testJobName);
+
+    buildAndDeployJobWithNoUser(dataJobDeploymentRequestBody);
+
+    buildAndDeployJob(dataJobDeploymentRequestBody);
+
+    buildAndDeployJobWrongTeam(dataJobDeploymentRequestBody);
+
+    buildAndDeployJobWithResources(testJobVersionSha);
+
+    verifyJobDeploymentResponse(testJobVersionSha);
+
+    executeJobDeploymentWithNoUser();
+
+    executeGetJobDeploymentWithWrongTeam();
+
+    executeDisableDeploymentNoUser();
+
+    executeDisableDeployment();
+
+    executeDisableDeploymentWrongTeam();
+
+    verifyDeploymentDisabled(jobDeploymentName);
 
-    // Take the job zip as byte array
+    verifyDeploymentFieldsAfterDisable(testJobVersionSha);
+
+    beforeDeploymentDeletion();
+
+    deleteDeploymentNoUser();
+
+    deleteDeploymentWrongTeam();
+
+    executeDeleteDeployment(jobDeploymentName);
+  }
+
+  @Test
+  @Order(4)
+  public void testDataJobDeleteSource() throws Exception {
```

Comment:
I don't see you deleting anything in this test?

Reply:
I didn't modify/refactor this test it's present in the original test file - https://github.com/vmware/versatile-data-kit/blob/df1ea41a89f493b12e6be56d73b64261aa214a2a/projects/control-service/projects/pipelines_control_service/src/integration-test/java/com/vmware/taurus/datajobs/it/BaseDataJobDeploymentCrudIT.java#L337-L350 
I'm not quite sure what it's supposed to do.

Reply:
deleted this test since it was redundant.